### PR TITLE
Aligned and added error messages #14875

### PR DESCRIPTION
**Pull request by a22maxra**
Old MOTD:
image

Now shows error messages which are aligned:
image
image

Old edit course messages:
image
Are now aligned and there an error for the Github Key field, which is optional:
image

Old new course messages:
image

Are now aligned:
image

### CODE REVIEW

**Review by a22willi — CHANGES_REQUESTED** [review]
Some changes requested:
- You have introduced mixed use of tabs and spaces in DuggaSys/courseed.js.
- The error messages fade in and out a lot. I counted 7 times after every change.

**Comment by a22maxra** [comment]
I fixed the spaces but the error messages fade in and out is from before since the validation gets called on each input which backs up the callback queue.

File: DuggaSys/courseed.js [hunk 1]
```diff
@@ -748,61 +748,51 @@ function setActiveCodes() {
...
+			messageElement.innerHTML = "A GitHub key should be 40 characters";
+		} else if (element.name === "coursecode") {
+			// Special handling for course codes
+			if (activeCodes.includes(element.value) && element.value !== element.dataset.origincode) {
+				messageElement.innerHTML = `${element.value} is already in use. Choose another.`;
+			} else {
+				messageElement.innerHTML = "2 Letters, 3 digits, 1 letter";
 		}
+		 }
 		return false;
```
Comment (a22willi) [inline DuggaSys/courseed.js:790]: Indentation.

**Review by a22willi — APPROVED** [review]
Just one small indentation fix, not enough to warrant another review, just fix it before you tell @a22moamy to merge.

**Pull request merged**